.github/workflows/run-tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
```

Comment:
We should also add an `enhancement` changelog entry for this. We can probably copy the wording from this PR: https://github.com/aws/aws-cli/pull/7063